.github/workflows/release-suite-native-preview-update.yml

```diff
@@ -0,0 +1,49 @@
+name: "[Release] suite-native preview update"
+
+on:
+  push:
+    branches: [develop]
+  pull_request:
+    paths:
+      - "suite-native/**"
+      - "suite-common/**"
+  workflow_dispatch:
+
+  jobs:
+    update:
+      if: github.repository == 'trezor/trezor-suite' || github.repository == 'trezor/trezor-suite-private'
+    name: EAS Update
+    environment: develop-suite-native
+    runs-on: ubuntu-latest
+    concurrency: fingerprint-${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+    env:
+      SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
```

Comment:
Are we sure we want to have sentry for the preview app builds?

Reply:
yes, there will be not much events from preview builds so it will not drain our quota, but it's usefull to see the errors early and be able to test changes related to sentry quicly